JSQMessagesViewController/Model/JSQVideoMediaItem.m

```diff
@@ -79,20 +97,51 @@ - (UIView *)mediaView
     if (self.fileURL == nil || !self.isReadyToPlay) {
         return nil;
     }
-    
-    if (self.cachedVideoImageView == nil) {
-        CGSize size = [self mediaViewDisplaySize];
-        UIImage *playIcon = [[UIImage jsq_defaultPlayImage] jsq_imageMaskedWithColor:[UIColor lightGrayColor]];
-        
-        UIImageView *imageView = [[UIImageView alloc] initWithImage:playIcon];
-        imageView.backgroundColor = [UIColor blackColor];
-        imageView.frame = CGRectMake(0.0f, 0.0f, size.width, size.height);
-        imageView.contentMode = UIViewContentModeCenter;
-        imageView.clipsToBounds = YES;
-        [JSQMessagesMediaViewBubbleImageMasker applyBubbleImageMaskToMediaView:imageView isOutgoing:self.appliesMediaViewMaskAsOutgoing];
-        self.cachedVideoImageView = imageView;
+
+    if (self.cachedVideoImageView == nil)
+    {
+        if (_image == nil)
+        {
+            CGSize size = [self mediaViewDisplaySize];
+            UIImage *playIcon = [[UIImage jsq_defaultPlayImage] jsq_imageMaskedWithColor:[UIColor lightGrayColor]];
+
+            UIImageView *imageView = [[UIImageView alloc] initWithImage:playIcon];
+            imageView.backgroundColor = [UIColor blackColor];
+            imageView.frame = CGRectMake(0.0f, 0.0f, size.width, size.height);
+            imageView.contentMode = UIViewContentModeCenter;
+            imageView.clipsToBounds = YES;
+            [JSQMessagesMediaViewBubbleImageMasker applyBubbleImageMaskToMediaView:imageView isOutgoing:self.appliesMediaViewMaskAsOutgoing];
+            self.cachedVideoImageView = imageView;
```

Comment:
let's pull this out into a new private helper method:

`configureVideoViewWithDefaultPlayIcon`